### PR TITLE
Information about capture operation inside step flow

### PR DESCRIPTION
**Pull request by christiansyoung**

### CODE REVIEW

File: vipps-checkout-api.md [hunk 1]
```diff
@@ -305,6 +305,17 @@ Vipps Checkout will expose a polling endpoint as described in our [Swagger](http
 
 In order to determine the status of a payment, you must poll epayment as described [here](https://vippsas.github.io/vipps-epayment-api/index.html#tag/QueryPayments/operation/getPayment).
 
+## Step 3a: If a transaction is authorized, capture payment
+
+If the state of session is `AUTHORISED`, the amount of the transaction was successfully _reserved_. In order to actually move the money and complete the payment, the transaction must be _captured_. If you ship physical goods, capture is not to be done until the shipment is sent to the customer. Otherwise, capture can happen anytime after the reservation happened as long as the customer has received what has been paid for. E.g. if you sell digital concert tickets and send them by email directly after a purchase.
```
Comment (eirikv) [inline vipps-checkout-api.md:310]: Maybe use "...until the shipment is _dispatched_ to the customer"?

**Review by eirikv — APPROVED** [review]

File: vipps-checkout-api.md (hunk 1 above)
Comment (alfmags) [inline vipps-checkout-api.md:310]: capture is not to be done until the shipment is sent to the customer -->

Capture must be done in accordance with you terms and conditions with the user. Our recommendation is that this is not performed until the goods or services have been delivered.
-----

Reason: Capture can still be performed prematurely in the case of pre-purchase where the customer has agreed to the T&C..

File: vipps-checkout-api.md [hunk 2]
```diff
@@ -305,6 +305,17 @@ Vipps Checkout will expose a polling endpoint as described in our [Swagger](http
...
+
+A capture operation for the whole amount reserved is called a "full capture". A "partial capture" is a capture operation on only parts of the amount reserved and is e.g. used if only parts of an order could be fulfilled. Captures cannot be made on a amounts larger than the reservation.
```
Comment (alfmags) [inline vipps-checkout-api.md:312]: on an amount*

**Review by alfmags — CHANGES_REQUESTED** [review]

File: vipps-checkout-api.md [hunk 3]
```diff
@@ -305,6 +305,17 @@ Vipps Checkout will expose a polling endpoint as described in our [Swagger](http
 
 In order to determine the status of a payment, you must poll epayment as described [here](https://vippsas.github.io/vipps-epayment-api/index.html#tag/QueryPayments/operation/getPayment).
 
+## Step 3a: If a transaction is authorized, capture payment
```
Comment (sebdak) [inline vipps-checkout-api.md:308]: Remember to update the table of contents with the new step, and remove the old one

File: vipps-checkout-api.md (hunk 2 above)
Comment (christiansyoung) [reply vipps-checkout-api.md]: 👍

File: vipps-checkout-api.md (hunk 1 above)
Comment (christiansyoung) [reply vipps-checkout-api.md]: Nice, swapped the whole thing with that sentence.

Comment (christiansyoung) [reply vipps-checkout-api.md]: Rewrote it

File: vipps-checkout-api.md (hunk 3 above)
Comment (christiansyoung) [reply vipps-checkout-api.md:309]: Done! And fixed a broken link there as well :)

**Review by alfmags — APPROVED** [review]
LGTM

File: vipps-checkout-api.md [hunk 4]
```diff
@@ -305,6 +306,17 @@ Vipps Checkout will expose a polling endpoint as described in our [Swagger](http
...
+
+If the state of session received in callback, or when polling, is `AUTHORISED`, the amount of the transaction was successfully _reserved_. In order to actually move the money and complete the payment, the transaction must be _captured_. Capture must be done in accordance with you terms and conditions with the user. Our recommendation is that this is not performed until the goods or services have been delivered.
```
Comment (SvenErikPy) [inline vipps-checkout-api.md:311]: you_r_ terms and conditions

Comment (christiansyoung) [reply vipps-checkout-api.md]: Fixed

**Pull request merged**